src/olympia/abuse/views.py

```diff
@@ -208,31 +208,32 @@ def filter_enforcement_actions(enforcement_actions, cinder_job):
 def process_webhook_payload_decision(payload):
     source = payload.get('source', {})
     log.info('Valid Payload from AMO queue: %s', payload)
-    if 'job' in source:
+    if source.get('decision').get('type') == 'api_decision':
+        log.debug('Cinder webhook decision for api decision skipped.')
+        raise CinderWebhookIgnoredError('Decision already handled via reviewer tools')
+    elif 'job' in source:
         job_id = source.get('job', {}).get('id', '')
 
         try:
             cinder_job = CinderJob.objects.get(job_id=job_id)
         except CinderJob.DoesNotExist as exc:
             log.debug('CinderJob instance not found for job id %s', job_id)
             raise CinderWebhookMissingIdError('No matching job id found') from exc
-    else:
-        decision_id = payload.get('previous_decision', {}).get('id', '')
-
+    elif prev_decision_id := payload.get('previous_decision', {}).get('id', ''):
         try:
-            decision = ContentDecision.objects.get(cinder_id=decision_id)
+            decision = ContentDecision.objects.get(cinder_id=prev_decision_id)
```

Comment:
For consistency now that you've renamed `decision_id` to `prev_decision_id`:
```suggestion
            prev_decision = ContentDecision.objects.get(cinder_id=prev_decision_id)
```

And then adjust below where it's used ?